src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -359,14 +359,14 @@ public void WindowsTrailingWhiteSpace()
         public void WindowsExtendedSyntaxWhiteSpace()
         {
             var paths = IOInputs.GetSimpleWhiteSpace();
-            using (TemporaryDirectory directory = new TemporaryDirectory())
-            {
-                foreach (var path in paths)
-                {
-                    string extendedPath = Path.Combine(IOInputs.ExtendedPrefix + directory.Path, path);
-                    Directory.CreateDirectory(extendedPath);
-                    Assert.True(Directory.Exists(extendedPath), extendedPath);
-                }
+
+            DirectoryInfo directory = Directory.CreateDirectory(GetTestFilePath());
```

Comment:
Yes for this one, no for the others (at least not without further changing the tests)